stories/components/Reply.stories.ts

```diff
@@ -0,0 +1,39 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Reply } from "@/components/reply";
+import { ReplyProps } from "@/components/reply/Reply";
+
+const ReplyMeta: Meta<ReplyProps> = {
+  title: "Components/Reply",
+  component: Reply,
+};
+
+export default ReplyMeta;
+
+export const Default: StoryObj<ReplyProps> = {
+  args: {
+    senderName: "User 1",
+    senderImage: "https://example.com/user1.jpg",
```

Comment:
These images do not render. You can reuse the one for Chatinput: https://i.ibb.co/gvynGqz/clipart1363971.png